test/pageutils/autocompleteitem.js

```diff
@@ -0,0 +1,14 @@
+import Item from "./item";
+
+class AutoCompleteItem extends Item {
+    constructor(locatorTemplates, itemNumber) {
+        super(locatorTemplates, itemNumber);
+        this.suggestResultLink = $(this.locators["suggestResultLink"]);
+    }
+
+    get suggestLink() {
+        return this.suggestResultLink;
```

Comment:
Why not 
```suggestion
        return $(this.locators["suggestResultLink"]);
```

Why the extra indirection?